src/data/nat/cast.lean

```diff
@@ -2,12 +2,23 @@
 Copyright (c) 2014 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
-
-Natural homomorphism from the natural numbers into a monoid with one.
 -/
 import algebra.ordered_field
 import data.nat.basic
 
+/-!
+# Cast of naturals
+
+This file defines the *canonical* homomorphism from the natural numbers into a monoid with one.
```

Comment:
`monoid with one` is misleading, since a `monoid` already has a one. Do you mean `add_monoid with one`?

Reply:
Oh yeah